.github/workflows/development.yml

```diff
@@ -38,6 +38,14 @@ jobs:
 
       - name: Run test cases and coverage
         run: yarn test:coverage
+        env:
+          REACT_APP_API_KEY: ${{ secrets.REACT_APP_API_KEY }}
```

Comment:
We will need integrate with other services. Can you name these variables appropriately to show that they're related to `firebase`.